utils/dump_leaderboard.go

```diff
@@ -114,25 +128,43 @@ func main() {
 		members = append(members, member)
 	}
 
+	if *sortFlag == "default" {
```

Comment:
```
switch *sortFlag {
  case "default":
    sort.Sort(ByName(members))
    break
  ...
}
```

Reply:
Ok done, and it [looks like I can skip](https://tour.golang.org/flowcontrol/9) `break`.